tests/varint/encoding.c

```diff
@@ -0,0 +1,34 @@
+#include "clar_libgit2.h"
+#include "varint.h"
+
+void test_varint_encoding__decode(void)
```

Comment:
I would actually move these into the `core` directory, so that they're in `tests/core/varint.c` and they become `tests_core_varint__encode` and `tests_core_varint__decode`.